go.mod

```diff
@@ -11,6 +11,7 @@ require (
 	github.com/codahale/hdrhistogram v0.0.0-20161010025455-3a0bb77429bd // indirect
 	github.com/fsnotify/fsnotify v1.4.9
 	github.com/gfleury/go-bitbucket-v1 v0.0.0-20200312180434-e5170e3280fb
+	github.com/go-enry/go-enry/v2 v2.8.0 // indirect
```

Comment:
I think you need to run `go mod tidy` here.

---

Side note: Have we thought about adding golang-ci-lint to this repo?

Reply:
I pushed a commit which does that. We can add golangci-lint, you are welcome to add it :)

Reply:
https://github.com/sourcegraph/zoekt/pull/218